### ISSUE TITLE
AttributeError with test `boundary_comp.py`

### ISSUE THREAD

**Issue by DanielKotik** [atomec-project/atoMEC#3]
`python boundary_comp.py` gives

```shell
func:'CalcEnergy' took: 83.0255 sec
Traceback (most recent call last):
  File "..../atoMEC_import/tests/boundary_comp.py", line 27, in <module>
    print("Total free energy with dirichlet :" + str(energy_dirichlet.F_tot))
AttributeError: 'dict' object has no attribute 'F_tot'
```

**Comment by timcallow** [comment]
This should now be fixed on fix_tests branch, please have a look and see PR #4

**Comment by DanielKotik** [comment]
I've merged PR #4. Close this now.
